VDA-XP-Reward/.env.example

```diff
@@ -0,0 +1,17 @@
+# Used to verify contract after deployment
+ETHERSCAN_API_KEY="<Input your etherscan.com api key>"
+POLYGONSCAN_API_KEY="<Input your polygonscan.com api key"
+
+# Used to deploy contract
+PRIVATE_KEY="<Input your wallet private key for deploying contract>"
+
+# Used for contract deployment & verification. Optional
+POLYGON_TESTNET_RPC="<Input your own RPC url for polygon mainet>"
+POLYGON_AMOY_RPC="<Input your own RPC url for amoy net>"
+POLYGON_MAINNET_RPC="<Input your own RPC url for mumbai net>"
+
+# Defender keys : used for multi-sign support scripts
+DEFENDER_TEAM_API_KEY="<Input your Defender Team API Key>"
```

Comment:
Are all of these required? If so, please explain why

Reply:
The `DEFENDER_...` keys are required to make proposals for following:
- Transfer the ownership to multi-sig wallet after the contract deployment
- Make a proposal for upgrading contract to newer version

As you know, we can't deploy the contract wit the multi-sig wallet. So, at the first, we deploy the contract with EOA account and then transfer the ownerships to the multi-sig wallet.
These `DEFENDER_...` keys are generated when we created the multi-sig wallet.

Reply:
Please add the explanation into the code or the README

Reply:
Added explain to the `.env.example` file